api/service/stagedstreamsync/syncing.go

```diff
@@ -38,41 +40,48 @@ var Buckets = []string{
 // CreateStagedSync creates an instance of staged sync
 func CreateStagedSync(ctx context.Context,
 	bc core.BlockChain,
+	consensus *consensus.Consensus,
 	dbDir string,
-	UseMemDB bool,
 	isBeaconNode bool,
 	protocol syncProtocol,
 	config Config,
 	logger zerolog.Logger,
-	logProgress bool,
 ) (*StagedStreamSync, error) {
 
-	isBeacon := bc.ShardID() == shard.BeaconChainShardID
-
+	if config.DebugMode {
+		fmt.Printf("[SSSD:CreateStagedSync] creating staged sync for shard:%d beaconNode:%t memdb:%t dbDir:%s serverOnly:%t minStreams:%d\n",
```

Comment:
seems to be a good log for utils.Logger().info() ? not sure if there are going to be a lot of more logs but it can't be worse than the p2p connected/disconnected logs right ?

Reply:
it will create much more logs than the p2p connection logs

Reply:
I think so